index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Linkbrary</title>
+  <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.8/dist/web/static/pretendard.css" />
+  <link rel="stylesheet" href="src/style.css">
+</head>
+<body>
+
+  <header>
+    <section>
+      <div class="header-bar">
+        <a href="/" class="link logo">
+          <img src="images/landing/logo.svg" alt="로고">
```

Comment:
셀프 클로징은 HTML5에서 권장 사항이긴 하지만 가독성을 위해서는 클로징이 들어가는 것을 권장합니다.

```suggestion
          <img src="images/landing/logo.svg" alt="로고" />
```